app/src/main/java/grinnell/appdev/edu/lyles/fragments/HotFoodFragment.java

```diff
@@ -3,28 +3,90 @@
 
 import android.os.Bundle;
 import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ListView;
 
+import org.json.JSONArray;
+
+import java.util.ArrayList;
+import java.util.concurrent.ExecutionException;
+
+import grinnell.appdev.edu.lyles.AsyncRetrieval;
+import grinnell.appdev.edu.lyles.ItemAdapter;
+import grinnell.appdev.edu.lyles.JSONConstants;
+import grinnell.appdev.edu.lyles.MenuItem;
 import grinnell.appdev.edu.lyles.R;
 
 /**
  * A fragment to display available hot food to user
  */
 public class HotFoodFragment extends Fragment {
 
+    private AsyncRetrieval mAsyncRetrieval;
+    private String mJsonBody;
+    private JSONArray mAllItemsAsJsonArray;
+
+    private ArrayList<MenuItem> mMenuItemArrayList;
+    private ItemAdapter mItemAdapter;
+    private RecyclerView mRecyclerView;
 
     public HotFoodFragment() {
         // Required empty public constructor
     }
 
-
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
```

Comment:
Let's extract the numerous events happening in `onCreateView()` into their own methods to keep it short.